packages/excel-builder-vanilla/src/interfaces.ts

```diff
@@ -140,3 +141,44 @@ export interface ExcelStyleInstruction {
   /** style id */
   style?: number;
 }
+
+// ---------------------------
+// Chart related interfaces
+// ---------------------------
+export type ChartType = 'bar' | 'line' | 'pie' | 'scatter';
```

Comment:
by looking at w3school most popular Excel Charts, we should rename `bar` to `column` (bottom-top) and then add `bar` as a left-right chart. Also might be a good idea to add stacked charts for both bar/column charts

https://www.w3schools.com/excel/excel_charts.php